RadixWallet/Features/AssetsFeature/AssetsView+View.swift

```diff
@@ -68,6 +68,9 @@ extension AssetsView {
 						}
 					}
 				}
+				#if !DEBUG
```

Comment:
In DEBUG the default value will be used, which is `false`. So Fiat value will always be shown in DEBUG mode.